packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -16,10 +25,20 @@ export function cloudflare<T extends Record<string, WorkerOptions>>(
 ): vite.Plugin {
 	let viteConfig: vite.ResolvedConfig;
 
+	/**
+	 * A set of worker/environment names that require Node.js compatibility.
+	 */
+	let requiresNodeCompat: Set<string>;
+
 	return {
 		name: 'vite-plugin-cloudflare',
 		config() {
 			return {
+				resolve: {
+					alias: { ...getNodeCompatAliases() },
```

Comment:
I did this to make it clear that we are creating a new alias object, in case people wanted to add other aliases.
I think Dario did something similar with externals elsewhere.
But yes. OK. I can flip it back.